package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
- [ ] There should be 2 files committed: a .html and .css file, kindly consider removing any other files

Reply:
I can not remove the other files. Please suggest.

Reply:
check this [tutorial](https://www.bing.com/ck/a?!&&p=88d9b5b2044f20506c70005b12b381425c0fc1a378e2f9c05431c844a231dab9JmltdHM9MTY1NjU3Mzg4NCZpZ3VpZD1iMjE4YzZkYi0wZjI2LTQ2YjMtODM1ZS00MjY4Njc4M2JhY2ImaW5zaWQ9NTE4Mg&ptn=3&fclid=b6c00adc-f845-11ec-bd8a-adec6b7d0394&u=a1aHR0cHM6Ly93d3cuYXRsYXNzaWFuLmNvbS9naXQvdHV0b3JpYWxzL3VuZG9pbmctY2hhbmdlcy9naXQtcm0&ntb=1) on how to remove files using git

Reply:
Also, next time add a description in your pull request. This should be the summary of what went into your project that you created the pull request for. Thanks